prompting/cleaners/all_cleaners.py

```diff
@@ -83,47 +85,57 @@ def apply(self, generation: str) -> str:
             input_string=generation
         )  # LLMs are good at being formal. Do the same if we remove a prefix.
 
+
 class PrunePostQuestionText(BaseCleaner):
     def __init__(self, **kwargs):
         pass
 
-    def apply(self, generation: str, min_pos: Union[int,float] = 5, max_pos: Union[int,float]= 0.5, max_questions: int = None) -> str:
-        
+    def apply(
+        self,
+        generation: str,
+        min_pos: Union[int, float] = 5,
+        max_pos: Union[int, float] = 0.5,
```

Comment:
Minor suggestion, starting from python `3.10` `Union` can be replaced with `|`
```suggestion
        min_pos: int | float = 5,
        max_pos: int | float = 0.5,
```